src/common/eeprom.h

```diff
@@ -18,11 +18,23 @@ enum {
 #define EEPROM_FEATURE_LAN     0x0004
 #define EEPROM_FEATURE_SHEETS  0x0008
 #define EEPROM_FEATURES        (EEPROM_FEATURE_PID_NOZ | EEPROM_FEATURE_PID_BED | EEPROM_FEATURE_LAN | EEPROM_FEATURE_SHEETS)
-
+#define DEFAULT_HOST_NAME      "PrusaMINI"
```

Comment:
This #define is not needed outside eeprom implementation.

Reply:
just like other defines above, wanted to have them at 1 place